aspnet/fundamentals/routing.rst

```diff
@@ -1,114 +1,297 @@
+        name: "default",
+        template: "{controller=Home}/{action=Index}/{id?}");
+
+This template will match a URL path like ``/Products/Details/17`` and extract the route values ``{ controller = Products, action = Details, id = 17 }``. The route values are determined by splitting the URL path into segments, and matching each segment with the *route parameter* name in the route template. Route parameters are named, and defined by enclosing the parameter name in braces ``{ }``.
+
+The template could also match the URL path ``/`` and would produce the values ``{ controller = Home, action = Index }``. This is allowed because the ``controller`` and ``{action}`` route parameters have default values, and the ``id`` route parameter is optional. An equals ``=`` sign followed by a value after the route parameter name defines a default value for the parameter. A question mark ``?`` after the route parameter name defines the parameter as optional. Route parameters with a default value *always* produce a route value when the route matches - optional parameters will not produce a route value if there was no corresponding URL path segment.
+
+See route-template-reference_ for a thorough description of route template features and syntax.
+
+This example includes a route constraint:
+
+.. code-block:: c#
+
+    routes.MapRoute(
+        name: "default",
+        template: "{controller=Home}/{action=Index}/{id:int}");
+
+This template will match a URL path like ``/Products/Details/17``, but not ``/Products/Details/Apples``. The route parameter definition ``{id:int}`` defines *route constraint* for the ``id`` route parameter. Route constraints implement ``IRouteConstraint`` and inspect route values to verify them. In this example the route value ``id`` must be convertable to an integer. See route-constraint-reference_ for a more detailed explaination of route constraints that are provided by the framework.
+
+Additional overloads of ``MapRoute`` accept values for ``constraints``, ``dataTokens``, and ``defaults``. These additional parameters of ``MapRoute`` are defined as type ``object``. The typical usage of these parameters is to pass an anonymously typed object, where the property names of the anonymous type match route parameter names.
+
+The following two examples create equivalent routes:
+
+.. code-block:: c#
+
+    routes.MapRoute(
+        name: "default_route",
+        template: "{controller}/{action}/{id?}",
+        defaults: new { controller = "Home", action = "Index" });
+
+    routes.MapRoute(
+        name: "default_route",
+        template: "{controller=Home}/{action=Index}/{id?}");
+
+.. tip:: The inline syntax for defining constraints and defaults can be more convenient for simple routes. However, there are features such as data tokens which are not supported by inline syntax.
+
+This example demonstrates a few more features:
+
+.. code-block:: c#
+
+    routes.MapRoute(
+        name: "blog",
+        template: "/Blog/{*article}",
```

Comment:
I changed it to 
``template: "{Blog}/{*article}",``

Is there something I'm missing where you wanted to show ``/``?